packages/ember-views/lib/mixins/view_support.js

```diff
@@ -610,6 +613,7 @@ export default Mixin.create({
     this.scheduledRevalidation = false;
```

Comment:
although not part of this commit, should this assignment to `this` be before the `init` ?